src/screens/Library/components/GameCard/index.tsx

```diff
@@ -198,80 +206,39 @@ const GameCard = ({
             to={{
               pathname: `/gameconfig/${appName}`
             }}
+            style={
+              { '--installing-effect': installingGrayscale } as CSSProperties
+            }
           >
-            <span
-              style={{
-                backgroundImage: `url('${
-                  grid ? cover : coverList
-                }?h=400&resize=1&w=300')`,
-                backgroundSize: '100% 100%',
-                filter: isInstalled ? 'none' : `grayscale(${effectPercent})`
-              }}
-              className={grid ? 'gameImg' : 'gameImgList'}
-            >
-              {logo && (
-                <img
-                  alt="logo"
-                  src={`${logo}?h=400&resize=1&w=300`}
-                  style={{
-                    filter: isInstalled ? 'none' : `grayscale(${effectPercent})`
-                  }}
-                  className="gameLogo"
-                />
+            <img src={imageSrc} className={imgClasses} alt="cover" />
```

Comment:
I moved from a span to an img so it handles dimensions with the element, that helps with the dimensions of the wrapper `a` tag